cmd/tidb-lightning-ctl/main.go

```diff
@@ -308,6 +314,26 @@ func checkpointDump(ctx context.Context, cfg *config.Config, dumpFolder string)
 	return nil
 }
 
+func getLocalStoringTables(ctx context.Context, cfg *config.Config) error {
+	cpdb, err := checkpoints.OpenCheckpointsDB(ctx, cfg)
```

Comment:
I think we should check the checkpoint table existence first, or if the checkpoint db does not exist, this call will create one, and it won't be clean up..

Reply:
BTW, if the `backend` is not local, we can directly return nil here